charts/aws-ebs-csi-driver/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Helm chart
 
+## v2.23.0
+* Manage DaemonSets enablement ([#1732](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1732), [@monicastanciu](https://github.com/monicastanciu))
```

Comment:
Can we call this something that makes it obvious what it does, like "Add `node.enableLinux` parameter"?